stress/shell/src/main/java/alluxio/stress/cli/StreamRegisterWorkerBench.java

```diff
@@ -99,10 +99,14 @@ public void prepare() throws Exception {
                     .build());
     mBlockMap = blockMap;
 
-    // Prepare these block IDs concurrently
-    LOG.info("Preparing blocks at the master");
-    RpcBenchPreparationUtils.prepareBlocksInMaster(blockMap, getPool(), mParameters.mConcurrency);
-    LOG.info("Created all blocks at the master");
+    // the preparation is done by the invoking client
+    // so skip preparation when running in job worker
+    if (!mBaseParameters.mDistributed) {
+      // Prepare these block IDs concurrently
+      LOG.info("Preparing blocks at the master");
+      RpcBenchPreparationUtils.prepareBlocksInMaster(blockMap);
```

Comment:
One thing I observe is, when we are preparing blocks in this method, it takes long to finish and hard to know the progress (in order to estimate how long the test will take). Could you add some more logs in this `prepareBlocksInMaster()` somewhere to track progress? I think it's possible if you have some printouts from the lambda you submit to the threadpool. Also it'll be great if there's a total count (like an AtomicInt that tracks total). Thx!

Reply:
Done.